packages/components/src/tabs/src/components/osds-tabs/core/controller.ts

```diff
@@ -54,9 +54,14 @@ class OdsTabsController {
 
   handleArrowKey(event: KeyboardEvent): void {
     const currentSelectedTabIndex = this.tabItems.findIndex((tab) => tab.hasAttribute('active'));
-    if(currentSelectedTabIndex === 0 && event.code === 'ArrowLeft' ) {
+    if (currentSelectedTabIndex === -1 || typeof currentSelectedTabIndex === 'undefined' ) {
```

Comment:
Why `typeof currentSelectedTabIndex === 'undefined'` ?

Reply:
To fix the issue that Damien reported in one of his comment

Reply:
issue is not about undefined, it's about any value that may cause `this.tabItems[currentSelectedTabIndex - 1]` to fails